sdk/template-dpg/Azure.ServiceTemplate.Template/.template.config/template.json

```diff
@@ -0,0 +1,194 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "chunyu@microsoft.com",
```

Comment:
This is PII .. do we require this?

Reply:
Do you mean if we need 'templage.json' file or do we need the two line 'schema' and 'author'?